Support encrypted system disk (#281)
gardener/gardener-extension-provider-alicloud: docs/usage-as-end-user.md

            "Resource": [
                "*"
            ]
        },
        {
            "Action": [
                "ros:*"
            ],
            "Effect": "Allow",
            "Resource": [

>                 "*"

Author: Not needed. It needs all access privilige.

## `WorkerConfig`

The Alicloud extension does not support a specific `WorkerConfig`. However, it supports additional data volumes (plus encryption) per machine.
By default (if not stated otherwise), all the disks are unencrypted.
For each data volume, you have to specify a name.
It also supports encrypted system disk. 

> However, only [Customized image](https://www.alibabacloud.com/help/doc-detail/172789.htm?spm=a2c63.l28256.b99.244.5da67453bNBrCt) is currently supported to used as a basic image for encrypted system disk.

Reviewer: ```suggestion
However, only [Customized image](https://www.alibabacloud.com/help/doc-detail/172789.htm?spm=a2c63.l28256.b99.244.5da67453bNBrCt) is currently supported to be used as a basic image for encrypted system disk.
```

      volume:
        size: 50Gi
        type: cloud_efficiency
        # NOTE: This comment is for the case when encrypted of an existing shoot is updated from false to true. 

>         # It will cause affected nodes rolling updated. It is must to MAINTAIN the shoot. 

Author: ```suggestion
        # It will cause affected nodes rolling updated. It must MAINTAIN the shoot. 
```
```suggestion
        # It will cause affected nodes rolling updated. It is must to MAINTAIN the shoot. 
```

>         # NOTE: This comment is for the case when encrypted of an existing shoot is updated from false to true. 
>         # It will cause affected nodes rolling updated. It is must to MAINTAIN the shoot. 
>         # Otherwise, the shoot will fail to reconcile.
>         # You could do it either via Dashboard or annotate the shoot with gardener.cloud/operation=maintain

Reviewer: ```suggestion
        # NOTE: Below comment is for the case when encrypted field of an existing shoot is updated from false to true. 
        # It will cause affected nodes to be rolling updated. Users must trigger a MAINTAIN operation of the shoot. 
        # Otherwise, the shoot will fail to reconcile.
        # You could do it either via Dashboard or annotating the shoot with gardener.cloud/operation=maintain
```
